modules/battery_manager/README.md

```diff
@@ -0,0 +1,53 @@
+# Battery Manager
+
+## Purpose
+
+The Battery Manager module allows users to **browse**, **add**, **edit**, and **activate/deactivate** entries 
+in the Test Battery. 
+
+## Intended Users
+
+The Battery Manager module is used by study administrators.
+
+## Scope
+
+The Battery Manager module provides a tool for browsing, adding, editing, and activating/deactivating entries in the
+the Test Battery.
+
+### Browse
+
+The **browse** functionality is under the `Browse` tab.
+Users can browse entries in the Test Battery and filter the entries using the `Selection Filters`.
```

Comment:
These paragraphs with line breaks after every sentence are weird and hard to read (also, some of them are too long. `fmt`.